docs/network/architecture.md

```diff
@@ -13,12 +13,27 @@ These components are instances of binaries in the [Octez software suite](https:/
 ## Sequencer
 
 Etherlink relies on a sequencer to publish transactions.
-It receives transactions from EVM nodes, puts them in order, and publishes them in two ways:
+It receives transactions from EVM nodes, puts them in order, and packages them into a blueprint, which is a group of Etherlink transactions.
 
-- It publishes transactions to EVM nodes, which can consider them final as long as they trust that the sequencer will publish them to layer 1.
-- It publishes transactions to Tezos layer 1, which finalizes them.
+Each blueprint contains:
+
+- A list of transactions
+- A list of transactions that are currently in the delayed inbox, as described in [Transaction lifecycle](#transaction-lifecycle)
+- The hash of the previous blueprint
+- The timestamp of the blueprint.
+
+You can imagine blueprints as the blocks in Etherlink's layer 2 chain.
+
+The sequencer publishes each blueprint in two ways:
+
+- It publishes them to EVM nodes, which can consider the transactions final as long as they trust that the sequencer will publish them to layer 1.
+- It publishes them to Tezos layer 1, which finalizes the transactions.
+
+The sequencer is the primary way that Etherlink transactions are processed.
+However, to protect the system from censorship and any other problems with the sequencer, Etherlink provides a backup way of handling transactions; see [Transaction lifecycle](#transaction-lifecycle).
 
 The sequencer is an instance of the `octez-evm-node` binary running in sequencer mode.
+Only members of the Sequencer Committee can run instances of the sequencer in sequencer mode.
```

Comment:
there's no committee yet, it might be confusing

Reply:
We talk about the Sequencer Committee in the governance info, so I'm comfortable using that term here. We could remove this sentence if it is confusing; it's just some extra info that I thought would be helpful.